### PR TITLE
Rubric for decentralized characteristics

### PR DESCRIPTION
**Pull request by brentzundel**
add requirement to charter for the WG to recommend a conformance rubric for method specs.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -129,10 +129,12 @@ <h1 id="title">Decentralized Identifier Working Group Charter</h1>
 
         <p>
 URL identifiers in use on the Web today (2019) require that the identifier is
```
Comment (msporny) [inline index.html]: This change shouldn't be in here. Looks like you branched from your other commit?

File: index.html [hunk 2]
```diff
@@ -237,6 +239,12 @@ <h2>Scope</h2>
 that are conformant with the data model and syntax(es).
           </li>
           <li>
+Recommend a conformance rubric of decentralized characteristics for DID Method
+specifications. This allows the DID Method specs to self-certify, or
+independent third parties to evaluate, the DID Method spec's level of
```
Comment (msporny) [inline index.html]: "spec's" -> "specification's

File: index.html [hunk 3]
```diff
@@ -237,6 +239,12 @@ <h2>Scope</h2>
 that are conformant with the data model and syntax(es).
           </li>
           <li>
+Recommend a conformance rubric of decentralized characteristics for DID Method
```
Comment (msporny) [inline index.html]: "conformance" means something special in standards speak. Suggest striking that word and use "Recommend a rubric of..." instead.

File: index.html [hunk 4]
```diff
@@ -237,6 +239,12 @@ <h2>Scope</h2>
...
+specifications. This allows the DID Method specs to self-certify, or
+independent third parties to evaluate, the DID Method spec's level of
+conformance.
```
Comment (msporny) [inline index.html]: Again, "conformance" is problematic. Alternate: "level of adherence to principles of decentralization".

**Review by msporny — CHANGES_REQUESTED** [review]
We need to be careful with "conformance" as that draws bright lines around what is and isn't allowed. The rubrics were supposed to not do that but rather say "things with characteristics of X, Y, and Z are more inline with the design intent of the specification than things with characteristics of only X and Y".

File: index.html (hunk 4 above)
Comment (brentzundel) [reply index.html]: addressed in [b3d5a0e](https://github.com/w3c-ccg/did-wg-charter/pull/28/commits/b3d5a0ecb1a19c3405195345fde41316901543f2)

File: index.html (hunk 3 above)
Comment (brentzundel) [reply index.html]: addressed in [b3d5a0e](https://github.com/w3c-ccg/did-wg-charter/pull/28/commits/b3d5a0ecb1a19c3405195345fde41316901543f2)

File: index.html (hunk 2 above)
Comment (brentzundel) [reply index.html]: addressed in [b3d5a0e](https://github.com/w3c-ccg/did-wg-charter/pull/28/commits/b3d5a0ecb1a19c3405195345fde41316901543f2)

File: index.html (hunk 1 above)
Comment (brentzundel) [reply index.html]: addressed in [1fd80c1](https://github.com/w3c-ccg/did-wg-charter/pull/28/commits/1fd80c1636204dba15ddcead66183ad4187b870c)

**Comment by nadalin** [comment]
@msporny Rubics are not normative unless they can have conformance, thus rubrics should not be in a charter

**Comment by msporny** [comment]
@nadalin wrote:
> Rubics are not normative unless they can have conformance, thus rubrics should not be in a charter

It is perfectly reasonable for a WG to develop NOTEs or sections in a specification that are non-normative. Charters can call out the sort of work product that a WG is expected to produce, such as a set of rubrics, even if it is non-normative.

**Comment by nadalin** [comment]
@msporny That would be part of a specification not a charter, if there is normative work this should be called out but I'm unaware of calling out non-normative work in a charter

**Comment by msporny** [comment]
@nadalin wrote:
>  That would be part of a specification not a charter, if there is normative work this should be called out but I'm unaware of calling out non-normative work in a charter

The Web Authentication Charter, which you co-Chair, calls out non-normative work in it's charter:

https://www.w3.org/Webauthn/charter/#ig-other-deliverables

**Comment by nadalin** [comment]
@msporny yes we did but that was POSIBLE work, please look at what we specified and we explicitly called these out as non-normative, these are, test suites, BCP, and protocol design. Not sure how rubrics would fit in, please explain.

**Comment by TallTed** [comment]
@msporny, @brentzundel - I think the issue @nadalin is seeing here lies in a blur between "scope" and "deliverables" in the current document draft. "Deliverables" definitely can include NOTEs (or non-normative sections of SPECs) which may include rubrics, all of which are implicitly within "scope" (which might then not need to be called out quite so explicitly).

**Comment by nadalin** [comment]
@TallTed Correct, please have a non-normative set of deliverables defined

**Comment by brentzundel** [comment]
@TallTed, @nadalin, @msporny - I added the rubric to the set of non-normative deliverables in [46de811](https://github.com/w3c-ccg/did-wg-charter/pull/28/commits/46de811fe648a038ffd0204911aaef3a4a7e4a04)

**Comment by brentzundel** [comment]
@msporny have my changes answered your concerns?

**Review by kimdhamilton — APPROVED** [review]
Latest changes (addressing @msporny's feedback) LGTM. Ok to merge IMO

**Comment by kimdhamilton** [comment]
> @msporny have my changes answered your concerns?

Hey @msporny your requested changes are blocking PR merge. @brentzundel's updates looked good; can you review/approve?

**Review by ken-ebert — APPROVED** [review]

**Review by msporny — APPROVED** [review]

**Review by msporny — APPROVED** [review]

**Pull request merged**